src/test/scala/core/RegisterFileTest.scala

```diff
@@ -25,10 +25,10 @@ class RegisterFileTest extends FlatSpec with ChiselScalatestTester {
   def registerFile = new RegisterFile(threads=threads)
 
   /* Write something to the register file */
-  def write(c: Module, w: RegisterFileWriteIO, thread: Int, addr: Int, data: UInt): Unit = {
+  def write(c: Module, w: RegisterFileWriteIO, enable: Bool, thread: Int, addr: Int, data: UInt): Unit = {
     require(addr < 32)
     timescope {
-      w.enable.poke(true.B)
+      w.enable.poke(enable)
```

Comment:
In addition to this we should probably also poke the default for enable to be `false.B`

Reply:
Sounds good. Will add that to the test.